docs/adr/00XX-political-and-government-tagging.md

```diff
@@ -0,0 +1,61 @@
+# XX. Political and Government Tagging
```

Comment:
Any reason why we haven't committed to a number yet?

Reply:
Yes - in the past these have sometimes dragged on for a while and the numbers have become conflicting.

Reply:
If I approve, will you edit the number before merging or does it happen in a separate PR?

Reply:
Yup before merging.